src/bin/contribute.rs

```diff
@@ -1011,19 +1052,24 @@ impl Contribute {
                     )
                 }
             };
-
+            warn!("Moving chunk {} to upload", chunk_id);
```

Comment:
Is this correct to be a `warn`? I'd think this is `debug` or `info`
Same about others

Reply:
I meant just to delete these, but looks like I missed them.